python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -1188,7 +1188,15 @@ def _create_prefetch_block(self, pserver_index, pserver_program,
     def _create_table_optimize_block(self, pserver_index, pserver_program,
                                      pre_block_idx, grad_to_block_id):
         # STEP: create table optimize block
+        table_opt_block = pserver_program._create_block(pre_block_idx)
         # create table param and grad var in pserver program
+        # create table optimize block in pserver program
+        table_opt_op = [
```

Comment:
why should put this before head

Reply:
No other reasons just make it formatted.